wagtail/wagtailadmin/static_src/wagtailadmin/js/blocks/sequence.js

```diff
@@ -89,7 +93,7 @@ CODE FOR SETTING UP SPECIFIC UI WIDGETS, SUCH AS DELETE BUTTONS OR MENUS, DOES N
         function setInitialMoveUpDownState(newMember) {
         }
 
-        function postInsertMember(newMember) {
+        self._postInsertMember = function(newMember) {
```

Comment:
What's the significance of the leading underscore here? I'd assume it was meant to indicate that this is an internal-only function rather than part of the `Sequence` API, but it seems that we're going ahead and accessing it within `ListBlock` anyway.

Reply:
Yes, mostly to signify that it's not a function you should call yourself. I'm not actually even calling it, just replacing it with a decorator like function. Which is why I need it exposed on the sequence object itself.

It's a weird way of doing it, but I couldn't think of a nicer way to do it given javascript's scoping mechanisms, but someone who is more js savvy may know a better method.